public/scss/app/_desktop.scss

```diff
@@ -39,6 +39,16 @@ body {
 
     &.card {
       width: 100%;
+
+      &.files {
+        height: 60vh;
+
+        ul.files {
+          margin: 0;
+          max-height: calc(100% - 50px);
+          overflow-y: scroll;
```

Comment:
Consider making it `auto` so that it becomes scrollable only when it reaches the max height.

Reply:
Changed to auto, the scrollbar doesn't appear anymore only when needed